tests/Compilation/positive/test060.juvix

```diff
@@ -17,21 +17,20 @@ type Pair (A B : Type) :=
     };
 
 mf : Pair (Pair Bool (List Nat)) (List Nat) → Bool
-  | mkPair@{fst := mkPair@{fst; snd := nil}; snd := zero :: _} := fst
+  | mkPair@{fst := mkPair@{fst; snd := nil};
+           snd := zero :: _} := fst
```

Comment:
can we refacotr the `align (braces items)` so that the second field is aligned with the first field instead of the opening `{`?

Reply:
Now it's consistent with how it works with named arguments. I think that if we were to change this, we should change both in a separate issue.